DEVELOPMENT.md

```diff
@@ -151,6 +151,83 @@ to start a debugger to which attach from your IDE.
    curl localhost:8081/apis/apps/v1/deployments
    ```
 
+## Database
+
+### Database requirements
+
+* psql
+    ```bash
+    $ sudo dnf install postgresql
+    $ psql --version
+    psql (PostgreSQL) 15.6
+    ```
+
+* Optional: A SQL client is recommended to interact easily with the database.
+  For example [DBeaver Community](https://dbeaver.io/). To configure DBeaver, check these 
+  [instructions](https://github.com/kubearchive/kubearchive/blob/main/database/README.md).
```

Comment:
Move this to the development resources in the "tools" section.

Reply:
Done